test/assignment.coffee

```diff
@@ -235,6 +235,105 @@ test "destructuring assignment against an expression", ->
   eq a, y
   eq b, z
 
+test "destructuring assignment with objects and splats: ES2015", ->
+  obj = {a: 1, b: 2, c: 3, d: 4, e: 5}
+  throws (-> CoffeeScript.compile "{a, r..., s...} = x"), null, "multiple rest elements are disallowed"
+  throws (-> CoffeeScript.compile "{a, r..., s..., b} = x"), null, "multiple rest elements are disallowed"
```

Comment:
Isn't it better to do throw tests in error_messages.coffee, so we know that we get the error location right?